<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog2</class>
 <widget class="QDialog" name="SecDialog2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>631</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollBar" name="verticalScrollBar">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>260</y>
     <width>21</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="stu">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="course">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="student_plus">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="course_plus">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>back_label</zorder>
  <zorder>listWidget</zorder>
  <zorder>verticalScrollBar</zorder>
  <zorder>stu</zorder>
  <zorder>home</zorder>
  <zorder>course</zorder>
  <zorder>student_plus</zorder>
  <zorder>course_plus</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
